hub/integrations/tests/test_pytorch.py

```diff
@@ -384,3 +380,23 @@ def test_pytorch_local_cache(ds):
         )
         pytorch_small_shuffle_helper(0, 16, dls)
         local_cache.clear()
+
+
+@requires_torch
+def test_groups(local_ds, compressed_image_paths):
+    img1 = hub.read(compressed_image_paths["jpeg"][0])
+    img2 = hub.read(compressed_image_paths["png"][0])
+    with local_ds:
+        local_ds.create_tensor(
+            "images/jpegs/cats", htype="image", sample_compression="jpeg"
+        )
+        local_ds.create_tensor(
+            "images/pngs/flowers", htype="image", sample_compression="png"
+        )
+        for _ in range(10):
+            local_ds.images.jpegs.cats.append(img1)
+            local_ds.images.pngs.flowers.append(img2)
+    dl = local_ds.pytorch()
+    for cat, flower in dl:
+        np.testing.assert_array_equal(cat[0], img1.array)
+        np.testing.assert_array_equal(flower[0], img2.array)
```

Comment:
shouldn't this test select a specific group to train on? i recommend adding another tensor that is apart of another group like "others/pngs/cats" and then selecting the "images" group and asserting that only 2 tensors are being iterated over

Reply:
Added test.